README.md

```diff
@@ -28,18 +28,77 @@ python manage.py db upgrade
 python run.py
 ```
 
-## v0 API endpoints available
+## Available v0 API endpoints
 
-| API endpoint                  | description                                                                |
-| ------------                  | -----------                                                                |
-| /api/v0/ros/systems           | Shows list of all systems from Host Inventory having a Performance Profile |
-| /api/v0/ros/systems/<host_id> | Shows the Performance Profile of the selected <host_id>                    |
-| /api/v0/ros/status            | Shows the status of the server                                             |
+### Request
+`GET /api/ros/v0/systems` Shows list of all systems from Host Inventory having a Performance Profile
 
-To run via API:
-```
-curl -v -H "x-rh-identity: <identity-header-value>" -H "Content-Type: application/json" http://localhost:8080/api/ros/v0/systems
-curl -v -H "x-rh-identity: <identity-header-value>" -H "Content-Type: application/json" http://localhost:8080/api/ros/v0/status
-curl -v -H "x-rh-identity: <identity-header-value>" -H "Content-Type: application/json" http://localhost:8080/api/ros/v0/systems/<host_id>
-```
-For local dev setup, please note that use the x-rh-identity header encoded from your account number, the one used while running `make insights-upload-data` and `make ros-upload-data` commands.
+    curl -v -H "x-rh-identity: <header_value>" -H "Content-Type: application/json" https://ci.cloud.redhat.com/api/ros/v0/systems
```

Comment:
If I am not wrong then I guess in CI env you will also need to pass username:password via `-u` option.

Reply:
I'd suggest using final production url in the examples: https://cloud.redhat.com/api/ros/v0/systems . Users trying out ci/qa/staging environments should be replacing it with their specific urls.

Reply:
Ok makes sense! So @patilsuraj767 would the request for prod require -u option too?

Reply:
@apuntamb I am not sure regarding the prod env.